corehq/apps/app_manager/static/app_manager/js/report-module.js

```diff
@@ -247,11 +247,14 @@ var ReportModule = (function () {
         var currentReports = options.currentReports || [];
         var availableReports = options.availableReports || [];
         var saveURL = options.saveURL;
+        _.defer(function(){
```

Comment:
Thanks @orangejenny. It wasn't working without it, but that's because the `changeSaveButton` method wasn't defined yet. Fixed here 8dd0b36